### PR TITLE
Project V.1. Implementation

### PR DESCRIPTION
**Pull request by huseyinbiyik**
You can find the features added for this Pull Request below. 🧩

  - A splash page with the app's name and links to the signup and log-in pages were created.
- Sign up and log in to pages
  - The user can register in the app with their full name, email, and password (all mandatory).
  - The user can log into the app using email and password.
  - If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).

- Home page (categories page)
  - When the user logs in, they are presented with the categories page.
  - For each category, the user can see their name, icon, and the total amount of all the transactions that belong to that category.
  - When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
  - There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

- Transactions page
  - For a given category, the list of transactions is presented, ordered by the most recent.
  - At the top of the page, the user can see the total amount for the category (sum of all of the amounts of the transactions in that category).
  - There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new category" page
  - The user fills out a form to create a new category, indicating their name and icon (both mandatory).
  - The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.

- "Add a new transaction" page
  - The user fills out a form to create a new transaction with:
    - name (mandatory)
    - amount (mandatory)
    - categories (mandatory at least one)
  - The user clicks (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
  - When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.
- Testing
  - Unit and integration tests for Group and Expenses page.

Thank you for your review in advance! 🤓

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
-# WhereMoney
\ No newline at end of file
+## Moneye
+
+> A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## Live Presentation
+[Watch My Presentation on Loom](https://www.loom.com/share/10d47905b9084e80ba5f057f7176c2a6)
+
```
Comment (SajjadAhmad14) [inline README.md:10]: Please deploy the app on Heroku or Netlify and add a live link to README.

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
```
Comment (SajjadAhmad14) [inline app/views/devise/registrations/new.html.erb]: The app should follow the design guidelines for the User login and registration page. Pages should use the same colors and fonts as defined in the [design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).

register

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Huseyin, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
The project has a presentation. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏
- [ ] The font family for headings and text should be the same as in the [design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/devise/registrations/new.html.erb (hunk 2 above)
Comment (huseyinbiyik) [reply app/views/devise/registrations/new.html.erb]: Hey Sajjad. 🖖 Thank you for the review and feedback. 
Unfortunately, Proxima Nova font family is not free, it is paid font face. I found Lato font similar to Proxima Nova and used it. I hope that's okay. I fixed the authentication page styles.

**Review by phelian23 — APPROVED** [review]
Hi @huseyinbiyik,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight! 👏 🟢
✔️ Good commit messages
✔️ Descriptive Readme file

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by huseyinbiyik** [comment]
Thank you for the review Olawufemi. I will merge it now! 🎉

**Pull request merged**